cranelift-simplejit/src/backend.rs

```diff
@@ -17,6 +17,10 @@ use target_lexicon::PointerWidth;
 #[cfg(windows)]
 use winapi;
 
+const EXECUTABLE_DATA_ALIGNMENT: u8 = 0x10;
+const WRITABLE_DATA_ALIGNMENT: u8 = 0x8;
```

Comment:
These two numbers appear somewhat magical. Why 0x10 and 0x8? If these are meant to be target-specific, we should have `TargetMachine` provide them.

Reply:
Quoting myself:

> It took all alignment constants from faerie. https://github.com/m4b/faerie/blob/49aac0ca5b49c6ef95a57a95d8a006ab73e998e5/src/elf.rs#L268-L276

Reply:
Indeed, but Faerie didn't invent these constants either. I think these should be good for x64 at least. Agreed it would be cleaner to have this in the TargetMarchine, but we can do this as a follow-up; @bjorn3, do you prefer to defer this, or do it now?

Reply:
I would prefer to defer this.

Reply:
Works fine for me, opened #758, thanks for answering.

Reply:
I would expect data alignment to always be decided by the caller, instead of passing in an option. At the least, using different values for writable data vs readable data seems strange.